data/themes/default/hypha.css

```diff
@@ -350,6 +347,44 @@ button: last-of-type{
  margin-right: 0;
 }
 
+.login-wrapper{
+	padding:15px;
+	background-color: #eeeeff;
+}
+
+.login-wrapper label{
+	display: block;
+	font-weight: bold;
+}
+
+.login-wrapper button,
+.login-wrapper [type="submit"],
+.login-wrapper [type="button"],
+.login-wrapper [type="reset"],
+.login-wrapper [type="cancel"]{
+	-webkit-appearance: none;
+   -moz-appearance: none;
+   appearance: none;
```

Comment:
There are mixed spaces/tabs here.

Reply:
Also, all this new CSS uses tabs or 3 spaces for indentation, where the rest of the file uses single spaces.

Reply:
fixed